bitnami/thanos/values.yaml

```diff
@@ -1785,14 +1788,9 @@ receive:
   ## Receive Configuration
   ## Specify content for receive hashring
   ##
-  config: |
-    [
-      {
-        "endpoints": [
-            "127.0.0.1:10901"
-        ]
-      }
-    ]
+  config:
+    - endpoints:
+      - "127.0.0.1:{{ .Values.receive.service.grpc.port }}"
```

Comment:
I'm a little concerned about this change.
Let's say I have my **values.yaml** already adapted specifying my custom configuration for Thanos Receiver, if I upgrade my release I can find an unexpected behaviour

Reply:
Oh, I just saw you covered this possibility in the `thanos.receive.config` macro

Reply:
Yep I did. 👍